apps/web/pages/contact.vue

```diff
@@ -1,29 +1,48 @@
 <template>
   <NuxtLayout name="default">
-    <!-- TODO: remove this code when the re-captcha/back-end call is working -->
-    <div v-if="true" class="h-[500px] flex items-center justify-center">
-      <h1>{{ $t('coming-soon') }}</h1>
-    </div>
-    <div v-else class="md:max-w-[677px] mx-auto px-4 pt-4 pb-20 md:px-0 md:mt-4">
+    <div class="md:max-w-[677px] mx-auto px-4 pt-4 pb-20 md:px-0 md:mt-4">
       <h1 class="font-bold mb-10 typography-headline-3 md:typography-headline-2">
         {{ $t('contact.contact') }}
```

Comment:
As ```t(...)``` is already available and used in our component, please use that instead of looking&fetching the global ```$t(...)```